### DISCUSSION TITLE
Performance drop ever since switching distros.

### DISCUSSION THREAD

**Discussion by NetworkSpirits** [vinegarhq/sober#85] [General]
Does anyone know why ever since I switched to Mint on Cinnamon, Sobers performance halved? On Pop-os with Gnome my performance was basically as it was on windows. Can this be fixed somehow, or is it just a distro/DE thing (which I doubt) 
Specs: Ryzen 5 5500, Rx 6600 xt, 16gb ram

**Comment by nathblade16** [comment]
you should check your graphics drivers on mint

**Comment by Amgsdfyhadfgybi** [comment]
same thing was happening to me, switched to kde plasma and performance is much better now

**Comment by Naymmmm** [comment]
Try switching to the linux-zen kernel, it is optimized for gaming.

**Comment by somesillydude69** [comment]
The obvious reason is because pop os kernel is kinda better than Linux mint one

**Comment by olifurz** [comment]


**Reply by nathblade16** [reply]
Instead of checking your not on wayland, if your on wayland check that its not using xwayland, xwayland will cause lag so make sure its not using it

**Comment by PushSL** [comment]


**Reply by nathblade16** [reply]
the last time I checked an AMD RX 6600 XT was not in fact made by nvidia
